src/packages/core/components/field-dropdown-list/field-dropdown-list.element.ts

```diff
@@ -0,0 +1,213 @@
+import type { UmbPropertyTypeModel } from '@umbraco-cms/backoffice/content-type';
+import { UmbDocumentTypeDetailRepository } from '@umbraco-cms/backoffice/document-type';
+import { UmbChangeEvent } from '@umbraco-cms/backoffice/event';
+import {
+	css,
+	html,
+	customElement,
+	property,
+	state,
+	repeat,
+	ifDefined,
+	nothing,
+	query,
+} from '@umbraco-cms/backoffice/external/lit';
+import type { UUIComboboxEvent, UUIComboboxElement } from '@umbraco-cms/backoffice/external/uui';
+import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
+import { UmbMediaTypeDetailRepository } from '@umbraco-cms/backoffice/media-type';
+import {
+	UMB_DOCUMENT_TYPE_PICKER_MODAL,
+	UMB_MEDIA_TYPE_PICKER_MODAL,
+	UMB_MODAL_MANAGER_CONTEXT,
+	type UmbModalManagerContext,
+} from '@umbraco-cms/backoffice/modal';
+
+interface FieldPickerValue {
+	alias: string;
+	label: string;
+}
```

Comment:
@loivsen Super, thanks! This makes it possible for me to reuse the component on the Collection data-type configuration. 🙌

Reply:
Yeah, let me know if there is other things I need to add for it to be useable all around!